mvp-spec/model/transmission-request.md

```diff
@@ -694,7 +694,7 @@ The base64 representation of a data signature
 
 <tr>
 <td>
-<b>receiver</b>
+receiver<br>(<i>optional</i>)
```

Comment:
This is unrelated to this PR: the model was not up to date on `main`.
(same below)